packages/lib/modules/pool/LbpDetail/LbpHeader/LbpHeaderStats.tsx

```diff
@@ -0,0 +1,64 @@
+import { Flex, Box } from '@chakra-ui/react'
+import { AnimatedNumber } from '@repo/lib/shared/components/other/AnimatedNumber'
+import Stat from '@repo/lib/shared/components/other/Stat'
+import { safeToNumber } from '@repo/lib/shared/utils/numbers'
+
+type StatItem = {
+  label: string
+  value?: string | number | undefined | null
+}
+
+const IMAGE_TRANSFORM_ARRAY = ['rotate(180deg) scale(2)', 'rotate(180deg)', 'scale(1)']
+
+export function LbpHeaderStats() {
+  const stats: StatItem[] = [
+    {
+      label: 'Funds raised',
```

Comment:
Does this mean we still don't have this info? Shouldn't we add a TODO if that is the case?